applications/nrf_desktop/configuration/nrf52840dk_nrf52840/pm_static_ZDebugMCUBootQSPI.yml

```diff
@@ -0,0 +1,42 @@
+mcuboot:
+  address: 0x0
+  size: 0xc000
+  region: flash_primary
+mcuboot_pad:
+  address: 0xc000
+  size: 0x200
+app:
+  address: 0xc200
+  size: 0xd3e00
+mcuboot_primary:
+  orig_span: &id001
+  - mcuboot_pad
+  - app
+  span: *id001
+  address: 0xc000
+  size: 0xd4000
+  region: flash_primary
+mcuboot_primary_app:
+  orig_span: &id002
+  - app
+  span: *id002
+  address: 0xc200
+  size: 0xd3e00
+mcuboot_scratch:
```

Comment:
MCUBoot no longer uses scratch partition

Reply:
I will get rid of it

Reply:
removed - sizes updated